docs-android/app/src/main/res/values-pl/styles.xml

```diff
@@ -0,0 +1,23 @@
+<resources>
```

Comment:
This file is unnecessary, Android will use the default file in the "values" directory.